src/Components/ErrorBoundary.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
```

Comment:
Leaning about this was awesome, and then implementing it and doing our own learning to do it was very cool and felt good! :D